server_src/constants/template.constants.js

```diff
@@ -0,0 +1,5 @@
+const noSpoolOptionTemplate = `<option value="0">No Spool Selected</option>`;
+
+module.exports = {
+  noSpoolOptionTemplate
+};
```

Comment:
note to self, shift this stuff client side

Reply:
Nice plan of attack: filter/split it out and move client side in one go?

Reply:
Yeah sounds good, will be easier once I've templated off the client side stuff.